examples/design/EU-DEMO/EUDEMO_reactor.py

```diff
@@ -28,17 +28,22 @@
 
 from bluemira.base.config import Configuration
```

Comment:
Can we make this file notebook convertible please

Reply:
Will be in next push (with some actual docs about what the design is doing 😮) - an interesting observation though is that since the bluemira custom printing writes to stderr, the notebook gets a red background for all of our printing, which perhaps isn't so pretty.

Reply:
Oo yeah, so it does. I think the only way to fix that is to go to proper class based logging. 
At the moment we have sort of functionalised the logging but maybe a class based approach would give us some more flexibility to for instance pipe info and debug to stdout. 
Also it might help with the case when after a flush is done there is no newline so the 'box' starts on the wring line for the next message

Reply:
Yea, I was in a quandary about how to handle writing the new line after a set of flushing has finished - having a class that has a 'flushing' state that can be switched off on the next print would solve that, and I suspect would help with a lot of other logging flexibility.